src/passport-saml/saml.ts

```diff
@@ -628,24 +654,24 @@ class SAML {
 
     const signature = signatures[0];
```

Comment:
`signatures` should be defined above on 648 instead of here or on line 657 using `as string`